samples/Makefile

```diff
@@ -1,9 +1,40 @@
+VERSION := $(shell grep seldonio/mlserver ../scheduler/Makefile | cut -d':' -f2)
+MLSERVER_VERSION ?= ${VERSION}
+MLSERVER_FOLDER = mlserver_${MLSERVER_VERSION}
+TRITON_VERSION=23-03
+TRITON_FOLDER=triton_${TRITON_VERSION}
 EXAMPLES := $(wildcard *.ipynb) $(wildcard examples/*/*.ipynb)
 EXAMPLES_MD := $(EXAMPLES:.ipynb=.md)
 
 .PHONY: all
 all: $(EXAMPLES_MD)
 
+.PHONY: train-all
+train-all:
+	make -C ./scripts/models train-all
+	make -C ./scripts/explainers train-all
+	make -C ./examples/image_classifier train
+	make -C ./examples/income_classifier train
+
+.PHONY: upload-all
+upload-all:
+	# Upload all artefacts to GCS
+	make -C ./scripts/models upload-all
+	make -C ./scripts/explainers upload-all
+	make -C ./examples/image_classifier upload_artifacts
+	make -C ./examples/income_classifier upload_artifacts
+	# Rename all references in notebooks, YAMLs and MD files
+	find . \
+		-type f \( -name '*.ipynb' -o -name '*.md' -o -name '*.yaml' \) \
+		-exec \
+			sed -i "s/gs:\/\/\(.*\)\/mlserver_.*\/\(.*\)/gs:\/\/\1\/$(MLSERVER_FOLDER)\/\2/g" \
+			{} \;
+	find . \
+		-type f \( -name '*.ipynb' -o -name '*.md' -o -name '*.yaml' \) \
+		-exec \
+			sed -i "s/gs:\/\/\(.*\)\/triton_.*\/\(.*\)/gs:\/\/\1\/$(TRITON_FOLDER)\/\2/g" \
```

Comment:
do we actually train anything that is related to triton version?

Reply:
Yes, there are some artefacts going to a `triton_*` folder, e.g.

https://github.com/SeldonIO/seldon-core/blob/f455281a431ad93d0e97ced42098b627ce490b1c/samples/scripts/models/Makefile#L91-L93